### PR TITLE
multimarkdown 6.4.0

### PR DESCRIPTION
**Pull request by lembacon**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/multimarkdown.rb [hunk 1]
```diff
@@ -19,6 +19,11 @@ class Multimarkdown < Formula
   conflicts_with "discount", :because => "both install `markdown` binaries"
 
   def install
+    # Reported upstream 2 Sep 2018 https://github.com/fletcher/MultiMarkdown-6/issues/142
```
Comment (lembacon) [inline Formula/multimarkdown.rb]: Just reported this upstream as fletcher/MultiMarkdown-6#142. cc @DomT4

**Review by DomT4 — APPROVED** [review]
Feel free to pull this with bottles as-is. Nice spot here 👍. Deployment targets are rapidly become not fun, heh.

**Comment by lembacon** [comment]
Probably we could just fix `-mmacosx-version-min` in `shims/super/cc`. 🤨

**Comment by DomT4** [comment]
We should, but it's difficult to do universally, and sometimes it's used legitimately 🙃. A lot of the problem isn't so much it existing but upstreams setting it to a version of macOS and then not keeping on top of macOS' development environment and consequently not realising before we do that it's becoming problematic.

**Pull request closed without merging**